frodo.js

```diff
@@ -1,52 +1,45 @@
 /**
```

Comment:
SCRIPT-The CSS rules here seem to be redundant. Can you please review and remove unnecessary ones?